### ISSUE TITLE
Unable to load the correct exception list file

### ISSUE THREAD

**Issue by Guthen** [Augaton/scp-hacking-device-reloaded#3]
I think there is a piece of the code that was forgotten to be updated concerning the file loading. This creates a bug where the exception list is not being set to the one previously saved.

By reading the [loading code](https://github.com/Augaton/scp-hacking-device-reloaded/blob/main/lua/guthscp/modules/hdevicereloaded/server.lua#L25-L26), we can see that the path of the file used is not the same that the one used by the [saving code](https://github.com/Augaton/scp-hacking-device-reloaded/blob/main/lua/guthscp/modules/hdevicereloaded/server.lua#L63). The directory used should be `guthscp` and not `guth_scp` (or the other way around while it's the same). 

I would advise to create and use a variable set to the correct path instead of literal strings to avoid this kind of natural errors.

By the way, if you want to save and load files from the `guthscp` directory, you can use the functions from the [`data`](https://github.com/Guthen/guthscpbase/blob/master/lua/guthscp/keter/core/shared/data.lua) library of the base.

**Comment by Augaton** [comment]
I think the problem has been fixed ?

**Comment by Guthen** [comment]
I have tested and a part of it is fixed but now there is a new big issue which is the file can not be saved if the directories are not created yet (which is never the case unless you do it manually). The only directory that is checked is `guthscp` (which is actually already done by the base), but the directories `guthscp/hdevicereloaded/<map_name>/` are never checked (by "check", I mean using `file.CreateDir`). This result in the data never being able to either save or load (since the file can not be created).

**Comment by Augaton** [comment]
I think i see, i need to create the directory before implementing some data ? (This is a thing that's i never done before so .. yeah)

**Comment by Guthen** [comment]
Yeah exactly, if the directories are not created, tying to create files (and so writing data) in these directories will result in a no operation (i.e. `file.Write` will silently fail)
